distribution/docs/src/main/resources/_contents/_spatial-contents/integrating-spatial-contents.adoc

```diff
@@ -1379,6 +1379,483 @@ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="urn:catalog:metacard"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+                   <ogc:Not>
+                       <ogc:PropertyIsLike wildCard="*" singleChar="#" escapeChar="!">
+                        <ogc:PropertyName>AnyText</ogc:PropertyName>
+                        <ogc:Literal>GeoJson</ogc:Literal>
+                       </ogc:PropertyIsLike>
+                   </ogc:Not>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query></csw:GetRecords>
+----
+A query for records with a property containg a value like "N#ar" AND NOT another property containing a property with a value like "GeoJson"
+[source,xml,linenums]
+----
+<csw:GetRecords xmlns:ows="http://www.opengis.net/ows" xmlns:ns2="http://www.w3.org/1999/xlink"
+               xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml"
+               xmlns:csw="http://www.opengis.net/cat/csw/2.0.2"
+               xmlns:ns6="http://www.w3.org/2001/SMIL20/"
+               xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dct="http://purl.org/dc/terms/"
+               xmlns:ns9="http://www.w3.org/2001/SMIL20/Language" resultType="results"
+               outputFormat="application/xml" outputSchema="urn:catalog:metacard"
+               startPosition="1" maxRecords="20" service="CSW" version="2.0.2">
+   <csw:Query typeNames="csw:Record">
+       <csw:ElementSetName>full</csw:ElementSetName>
+       <csw:Constraint version="1.1.0">
+           <ogc:Filter>
+               <ogc:And>
+                       <ogc:PropertyIsLike wildCard="*" singleChar="#" escapeChar="!">
+                           <ogc:PropertyName>AnyText</ogc:PropertyName>
+                           <ogc:Literal>N#ar</ogc:Literal>
+                       </ogc:PropertyIsLike>
+                   <ogc:Not>
+                       <ogc:PropertyIsLike wildCard="*" singleChar="#" escapeChar="!">
+                        <ogc:PropertyName>AnyText</ogc:PropertyName>
+                        <ogc:Literal>GeoJson</ogc:Literal>
+                       </ogc:PropertyIsLike>
+                   </ogc:Not>
+               </ogc:And>
+           </ogc:Filter>
+       </csw:Constraint>
+   </csw:Query></csw:GetRecords>
+----
+===== `Output Format Examples`
+If an output type format is specified, the results will be transformed into the given format. The list of supported output formats can be found by doing a GetCapabilities request to the CSW endpoint. Supported output formats include:
```

Comment:
I know @roelens8 implemented product retrieval for csw but I'm not sure if it was documented.  Is it worth documenting the process to retrieve a product in this section?